src/IndexedDims.jl

```diff
@@ -40,7 +41,7 @@ function Base.setindex!(arr::IndexedDimsArray{T, N}, val, inds::Vararg{Int, N})
 end
 ###
 
-function Base.similar(arr::IndexedDimsArray{T, N, A}, inds::Tuple{Vararg{AbstractUnitRange, N}}) where {T, N, A}
+function Base.similar(arr::IndexedDimsArray{T, N, A}, inds::Tuple{Vararg{Union{Int, AbstractUnitRange}, N}}) where {T, N, A}
```

Comment:
Thius needs to move to  some other file

Reply:
yes, i will do this in a follow-up PR so as to not move cod whilst also editting code https://github.com/invenia/IndexedDims.jl/issues/17